dev_guide/routes.adoc

```diff
@@ -225,20 +225,42 @@ ifdef::openshift-online[]
 [[custom-route-and-hosts-and-certificates-restrictions]]
 == Restrictions
 
-Custom route hosts are not permitted. Instead, the following host template is
-enforced on all user routes:
+Routes are restricted in {product-title} Starter, but are not restricted in
+{product-title} Pro. Custom route hosts are permitted in {product-title} Pro. If
+using {product-title} Starter, the following host template is enforced on all
+user routes:
+
+----
+<route-name>-<namespace>.<external-address>
+----
+
+For example:
 
 ----
 <route-name>-<namespace>.1d35.starter-us-east-1.openshiftapps.com
 ----
 
-Also, custom certificates are not permitted. Only unencrypted routes, edge
-routes using the default certificate, and passthrough routes work. Edge routes
-with custom certificates and re-encrypt routes (which necessarily have custom
-certificates) do not work.
+To determine the external address, run:
+
+----
+$ oc get route/<route-name>
+----
+
+Custom certificates are permitted in {product-title} Pro. In {product-title}
+Starter, only unencrypted routes, edge routes using the default certificate, and
+passthrough routes work. Edge routes with custom certificates and re-encrypt
+routes (which necessarily have custom certificates) do not work in
+{product-title} Starter.
 
 These restrictions are enforced in the router. Inspecting the route in the
 console or using `oc get routes` displays the host and certificates specified by
```

Comment:
These two statements are no longer true.  The restriction will be enforced by the API server, which will reject the custom route if it violates restrictions on custom routes (which means there will be no route for `oc get routes` to display).  I can make a separate PR or Bugzilla report if you prefer that over overloading this PR.

Reply:
Correction: There will be a route for `oc get routes` to display, but its status will indicate if the route was restricted.